frontend/src/Explorer/View/Common.purs

```diff
@@ -295,8 +295,8 @@ noData = "--"
 currencyCSSClass :: Maybe CCurrency -> String
 currencyCSSClass mCurrency =
   case mCurrency of
-      Just ADA -> " ada bg-ada-dark"
-      Just USD -> " usd bg-usd-dark"
+      Just ADA -> " currency ada bg-ada-dark"
+      Just USD -> " currency usd bg-usd-dark"
```

Comment:
@akegalj Agree. I've already started with it to have more typed CSS. Here are the first steps (borrowed from `vending-application` or `rscoin  block explorer`, I can't remember ... ) https://github.com/input-output-hk/cardano-sl-explorer/blob/master/frontend/src/Explorer/View/CSS.purs